src/routes/map/europe/+page.svelte

```diff
@@ -1,155 +1,139 @@
 <script lang="ts">
 	import { onMount } from 'svelte';
 	import { goto } from '$app/navigation';
-	import type { PageData } from '../$types';
-	import CirclesEurope from '$lib/circle-components/Circles-europe.svelte';
-	import CirclesFull from '$lib/circle-components/Circles-full.svelte';
 	import { base } from '$app/paths';
-	import { colors } from '$lib/metadata.json';
+	import CirclesEurope from '$lib/circle-components/Circles-europe.svelte';
+	import colors from '$lib/colors.json';
+	import FullMap from '../FullMap.svelte';
 
-	let { data }: { data: PageData } = $props();
-	let color = $state();
+	let isPulsating = $state(false);
+	setTimeout(() => {
+		isPulsating = true;
+	}, 2000);
 
 	// Style circles
-	function applyStyles(el, color, url) {
-		el?.classList.add('marked');
-		el?.style.setProperty('r', '8px');
-		el?.style.setProperty('fill', `var(--color-${color}-500)`);
-		// el?.style.setProperty('stroke', `var(--color-${color}-600)`);
-		// el?.style.setProperty('stroke-width', 2);
-		el?.addEventListener('click', () => {
-			goto(url);
+	function applyStyles(elements, color, url) {
+		elements.forEach((el) => {
+			el?.style.setProperty('r', '8px');
+			el?.style.setProperty('fill', `var(--color-${color}-500)`);
+			el?.addEventListener('click', () => {
+				goto(url);
+			});
 		});
 	}
 
 	onMount(() => {
+		applyStyles(document.querySelectorAll('.europe-biel'), 'error', '{base}/timeline/Brunnen/1798');
```

Comment:
the links on the dots are wrong.
```suggestion
		applyStyles(document.querySelectorAll('.europe-biel'), 'error', `${base}/timeline/Brunnen/1798`);
```